skiko/src/nativeJsMain/kotlin/org/jetbrains/skia/impl/Native.nativejs.kt

```diff
@@ -0,0 +1,12 @@
+package org.jetbrains.skia.impl
+
+/**
+ * Converts String to zero-terminated utf-8 byte array.
+ */
+internal fun convertToZeroTerminatedString(string: String): ByteArray {
```

Comment:
The name is confusing - I thought that `toInterop` is recursive. It converts to `ByteArray`, not string

Also better to include "utf8" in some form to the name